doozer

```diff
@@ -1590,6 +1675,40 @@ installonly_limit=3
         green_print('All repos synced to {}'.format(output))
 
 
+@cli.command("config:update-required", short_help="Update images that are required")
+@click.option("--image-list", help="File with list of images, one per line.", required=True)
+@pass_runtime
+def config_update_required(runtime, image_list):
+    """Ingest list of images and update data repo
+    with which images are required and which are not.
+    """
+    _fix_runtime_mode(runtime)
+    runtime.initialize(**CONFIG_RUNTIME_OPTS)
+
+    with open(image_list, 'r') as il:
+        image_list = il.readlines()
+        image_list = [i.strip() for i in image_list if i.strip()]
+
+    found = []
+    count = 0
```

Comment:
looks below like `count == len(found)`, what am I missing?

Reply:
Ignore this... it's just me playing around. Finishing this today most likely.